firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -902,15 +886,7 @@ void loop()
     check_rx_timeouts();
         
     // check pressures on master cylinder (pressure from pedal)
-    smc.checkPedal();
-
-    // read and parse incoming serial commands
-    if( Serial.available() > 0 ) 
-    {
-        incomingSerialByte = Serial.read();
-        processSerialByte();
-    }
-      
+    //smc.checkPedal();
```

Comment:
This needs a comment explaining why it's commented out, or it needs to be removed.